gradio/flagging.py

```diff
@@ -127,8 +127,8 @@ def image_classifier(inp):
     Guides: using-flagging
     """
 
-    def __init__(self):
-        pass
+    def __init__(self, simplify_files: bool = True):
```

Comment:
We add this parameter because example caching also uses this method, and there, we actually do want to keep the full file data